superset-frontend/src/hooks/useIsEmbedded.ts

```diff
@@ -0,0 +1,5 @@
+import { useSelector } from 'react-redux';
+import { RootState } from 'src/dashboard/types';
+
+export const useIsEmbedded = () =>
+  useSelector<RootState, boolean>(({ dashboardInfo }) => !dashboardInfo.userId);
```

Comment:
I am not sure that this is a reliable way to identify an embedded context

Reply:
Hi @geido @sadpandajoe PHAL :)